app/views/recommendations/_party_recommendation.html.haml

```diff
@@ -2,9 +2,9 @@
   - if rec.match != :unknown
     - if rec.match == :good
       &#x2705
-    = link_to URI.parse(rec.site.link).host, rec.site.link
+    = link_to rec.site.name, rec.site.link, :title => "#{rec.site.name} say: #{rec.site.meta_desc}"
```

Comment:
Micro-nitpick: I think `says` would be more grammatically correct than `say` here, since it's referring to the site (singular), rather than the group of people (plural) behind the site?